.circleci/cimodel/data/caffe2_build_data.py

```diff
@@ -27,15 +28,24 @@ def get_children(self):
         return [self.child_constructor()(self, k, v) for (k, v) in self.subtree]
 
     def is_build_only(self):
-        if str(self.find_prop("language_version")) == "onnx_py3.6":
+        if str(self.find_prop("language_version")) == "onnx_py3.6_part1" or \
+                str(self.find_prop("language_version")) == "onnx_py3.6_part2":
             return False
+        if str(self.find_prop("language_version")) == "onnx_py3.6":
```

Comment:
I think why we have failures in ci, because you set it build only.
Build only ci won't upload the docker after the build stage...

So let's keep False, and I don't think we need is_test_only.

Reply:
So, as commented before: https://github.com/pytorch/pytorch/pull/33242#discussion_r379622457
If we want to keep a single build, that is used by two tests, the only way I can imagine is to have is_test_only parameter. Let me know if you think we can change this.